src/main/java/picard/sam/SamErrorMetric/CollectSamErrorMetrics.java

```diff
@@ -137,7 +137,7 @@
     public ReadBaseStratification.Stratifier STRATIFIER_VALUE;
 
     @Argument(shortName = "V", doc = "VCF of known variation for sample. program will skip over polymorphic sites in this VCF and " +
-            "avoid collecting data on these loci.")
+            "avoid collecting data on these loci.", optional = true)
```

Comment:
hmmm.  Collecting error metrics over variants doesn't make much sense...you could give it dbSNP or something like that if you dont' have truth for that sample....

Reply:
What if my sample has no variants relative the reference? That’s my real use case! No VCF is required. Think sequencing bacteria or synthetic sequences.